src/cljs/athens/views/athena.cljs

```diff
@@ -165,18 +164,16 @@
 
 (defn create-search-handler
   [state]
-  (debounce (fn [query]
-              (if (str/blank? query)
-                (reset! state {:index   0
-                               :query   nil
-                               :results []})
-                (reset! state {:index   0
-                               :query   query
-                               :results (->> (concat [(search-exact-node-title query)]
-                                                     (search-in-node-title query 20 true)
-                                                     (search-in-block-content query))
-                                             vec)})))
-            1000))
+  (fn [query]
+    (if (str/blank? query)
+      (reset! state {:index   0
+                     :query   nil
+                     :results []})
+      (reset! state {:index   0
+                     :query   query
+                     :results (->> (into (search-in-block-content query)
```

Comment:
we can write this better
concat is more idiomatic here and we can ditch the ->> from old since it's only one operation on data 

(vec (concat [(exact-node-title)] (node-title..) ))

Reply:
Updated